node/resolvers/Mutations/Organizations.ts

```diff
@@ -456,13 +457,13 @@ const Organizations = {
                   ...(defaultCostCenter.customFields && {
                     customFields: defaultCostCenter.customFields,
                   }),
+                  stateRegistration: defaultCostCenter.stateRegistration,
```

Comment:
So the main fix was to include `stateRegistration` in the `updateOrganizationRequest` mutation when creating a cost center and add unit tests to check that it's there?

`stateRegistration` is optional like `businessDocument` and it's value can be `null` so the pattern to include them was to check for the value first (see below), but I'm not seeing any issues when simply doing `stateRegistration: defaultCostCenter.stateRegistration`. 🤔  In both master and dev workspaces, it looks like `stateRegistration` is saved as an empty string instead of `null` when it's left blank.

```
...(defaultCostCenter.businessDocument && {
    businessDocument: defaultCostCenter.businessDocument,
})
```

Reply:
Good point, @AnnaChiu95, It's probably safer not do send the value at all if it's not set, just to be sure.

Reply:
Yes @AnnaChiu95 the change just was it. The another is because of sorting alphabetically the parameter, I got in my IDE and fix it.

I can make a test to validate this point, but looking the [`createOrganization`](https://github.com/vtex-apps/b2b-organizations-graphql/blob/hotfix/B2BTEAM-1191-state-registration/node/resolvers/Mutations/Organizations.ts#L86) implementation, the parameter is validated, so it is unnecessary to do again.